Mathlib/LinearAlgebra/BilinearForm/Hom.lean

```diff
@@ -305,6 +306,15 @@ theorem comp_congr (e : M' ≃ₗ[R] M'') (B : BilinForm R M) (l r : M' →ₗ[R
       B.comp (l.comp (e.symm : M'' →ₗ[R] M')) (r.comp (e.symm : M'' →ₗ[R] M')) :=
   rfl
 
+variable {N₁ N₂ : Type*}
+variable [AddCommMonoid N₁] [AddCommMonoid N₂] [Module R N₁] [Module R N₂]
+
+/-- When `N₁` and `N₂` are equivalent, bilinear maps on `M` into `N₁` are equivalent to bilinear
+maps into `N₂`. -/
+@[simps!]
+def _root_.LinearMap.BilinMap.congr₂ (e : N₁ ≃ₗ[R] N₂) : BilinMap R M N₁ ≃ₗ[R] BilinMap R M N₂ :=
```

Comment:
Do we have a the non-equiv spelling of this? If so, a lemma linking then would be good.

(In the quadratic case `simps` I believe generates such a lemma, but I will check more closely when I'm next at lean)

Reply:
`@[simps!?]` outputs:
```
[simps.verbose] adding projection LinearMap.BilinForm.LinearMap.BilinMap.congr₂_symm_apply_apply_apply:
      ∀ {R : Type u_1} {M : Type u_2} [inst : CommSemiring R] [inst_1 : AddCommMonoid M] [inst_2 : Module R M]
        {N₁ : Type u_9} {N₂ : Type u_10} [inst_3 : AddCommMonoid N₁] [inst_4 : AddCommMonoid N₂] [inst_5 : Module R N₁]
        [inst_6 : Module R N₂] (e : N₁ ≃ₗ[R] N₂) (f : M →ₗ[R] M →ₗ[R] N₂) (a a_1 : M),
        (((LinearMap.BilinForm.LinearMap.BilinMap.congr₂ e).symm f) a) a_1 = e.symm ((f a) a_1) 

[simps.verbose] adding projection LinearMap.BilinForm.LinearMap.BilinMap.congr₂_apply_apply_apply:
      ∀ {R : Type u_1} {M : Type u_2} [inst : CommSemiring R] [inst_1 : AddCommMonoid M] [inst_2 : Module R M]
        {N₁ : Type u_9} {N₂ : Type u_10} [inst_3 : AddCommMonoid N₁] [inst_4 : AddCommMonoid N₂] [inst_5 : Module R N₁]
        [inst_6 : Module R N₂] (e : N₁ ≃ₗ[R] N₂) (f : M →ₗ[R] M →ₗ[R] N₁) (a a_1 : M),
        (((LinearMap.BilinForm.LinearMap.BilinMap.congr₂ e) f) a) a_1 = e ((f a) a_1) 
```

Reply:
Right, that's what I was afraid of. I added the missing lemma.